rustls/src/crypto/aws_lc_rs/ticketer.rs

```diff
@@ -14,7 +14,7 @@ use aws_lc_rs::{hmac, iv};
 use super::ring_like::rand::{SecureRandom, SystemRandom};
 use super::unspecified_err;
 use crate::error::Error;
-#[cfg(feature = "logging")]
+#[cfg(debug_assertions)]
 use crate::log::debug;
```

Comment:
```suggestion
```

I think it will still work in all cases if we remove these lines here & in `rustls/src/crypto/ring/ticketer.rs`, unforuntaely do not know how to explain why. I did try removing the `use crate::log::` lines from some other modules, did not seem to work with logging enabled. I would have to investigate further to explain this.